crates/sound/src/audio_backends/pipewire.rs

```diff
@@ -245,7 +245,7 @@ impl AudioBackend for PwBackend {
             let mut stream_hash = self.stream_hash.write().unwrap();
             let mut stream_listener = self.stream_listener.write().unwrap();
             self.thread_loop.lock();
-            let stream_params = self.stream_params.read().unwrap();
+            let stream_params = self.stream_params.write().unwrap();
```

Comment:
Why we need a `write` lock now?

Reply:
We do not need a `write` lock. I'll change it back to a `read` lock.